config/prow/cluster/build_kubernetes-external-secrets_customresource.yaml

```diff
@@ -16,19 +16,6 @@ spec:
 ---
 apiVersion: kubernetes-client.io/v1
 kind: ExternalSecret
-metadata:
-  name: gcp-secrets-store-cred
-  namespace: test-pods
-spec:
-  backendType: gcpSecretsManager
-  projectId: secretmanager-csi-build
-  data:
-  - key: k8s-oss-prow
-    name: key.json
-    version: latest
----
-apiVersion: kubernetes-client.io/v1
-kind: ExternalSecret
 metadata:
   name: azure-secrets-store-cred
```

Comment:
I see that this secret also started to fail being synced, is this also supposed to be gone?

Reply:
@tam7t

Reply:
The `azure-secrets-store-cred` one should still succeed as it is  still in use @aramase

Reply:
@chaodaiG Which secret is being failed to sync? `azure-secrets-store-cred` or `gcp-secrets-store-cred`?

Reply:
Also, if it's `azure-secrets-store-cred` what are the errors when trying to sync?

Reply:
both these two secrets failed to sync with the same reason:

```
PERMISSION_DENIED: Permission 'secretmanager.versions.access' denied for resource 'projects/kubernetes-upstream/secrets/azure-secrets-store-cred/versions/latest' (or it may not exist).
```

Reply:
I just granted `kubernetes-external-secrets-sa@k8s-prow-builds.iam.gserviceaccount.com` the `Secret Manager Secret Accessor` and `Secret Manager Viewer` permissions. Let me know if the error resolves.

Reply:
it's succeeding now

Reply:
Thanks for confirming!